src/commands/info.js

```diff
@@ -13,19 +13,51 @@ governing permissions and limitations under the License.
 const {Command, flags} = require('@oclif/command')
 const envinfo = require('envinfo')
 
+const NEW_LINE = '\n'
+const INDENT = (n) => '  '.repeat(n)
+const AIO_PLUGINS = 'AioPlugins'
 
 class InfoCommand extends Command {
+
+
   async run() {
     const { flags } = this.parse(InfoCommand)
-    const resInfo = await envinfo.run({
+
+    try {
+      this.resInfo = await envinfo.run({
```

Comment:
We shouldn't set an instance variable. I don't think  we should have a dependency or side effect on an instance variable which is transient. This may cause bugs in the future.

I think the result from envinfo.run should be passed to getAioPluginsInfo as an input, and the final input returned from getAioPluginsInfo be logged instead.